docs/guides/integration/github.md

```diff
@@ -194,22 +109,46 @@ Set the
 option to use the pinned version for the project:
 
 ```yaml title="example.yml"
-steps:
-  - name: "Set up Python"
-    uses: actions/setup-python@v5
-    with:
-      python-version-file: ".python-version"
+name: Example
+
+jobs:
+  uv-example:
+    name: python
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v1
+
+      - name: "Set up Python"
+        uses: actions/setup-python@v5
+        with:
+          python-version-file: ".python-version"
```

Comment:
I find it useful to include the full example since it allows users to copy-paste. Any objections?

Reply:
If you'll be repeating the full example you should use `hl_lines` to highlight the change.

Reply:
You're a genius...